pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -183,7 +183,6 @@ func (configgen *ConfigGeneratorImpl) buildOutboundClusters(env *model.Environme
 				applyTrafficPolicy(opts)
 				defaultCluster.Metadata = util.BuildConfigInfoMetadata(destRule.ConfigMeta)
 				for _, subset := range destinationRule.Subsets {
-					inputParams.Subset = subset.Name
```

Comment:
why are you removing this?

Reply:
This field is useless, should be a legacy one.